packages/polaris-viz/src/components/LineChart/Chart.tsx

```diff
@@ -424,6 +430,10 @@ export function Chart({
           data={legend}
           onDimensionChange={setLegendDimensions}
           renderLegendContent={renderLegendContent}
+          renderHiddenLegendLabel={renderHiddenLegendLabel}
+          width={width}
+          dimensions={dimensions}
+          enableHideOverflow
```

Comment:
We talked about enabling this as the default, however do we still want to allow the option for a consumer to override the default? i.e make `enableHideOverflow` a prop on the charts?